tmva/tmva/test/DNN/RNN/TestForwardPass.cxx

```diff
@@ -2,15 +2,15 @@
 // Author: Saurav Shekhar 22/06/17
 
 /*************************************************************************
- * Copyright (C) 2017, Saurav Shekhar                                    *
- * All rights reserved.                                                  *
- *                                                                       *
- * For the licensing terms see $ROOTSYS/LICENSE.                         *
- * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ * Copyright (C) 2017, Saurav Shekhar                                                      *
+ * All rights reserved.                                                                          *
+ *                                                                                                        *
+ * For the licensing terms see $ROOTSYS/LICENSE.                                     *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.                   *
```

Comment:
The changes above are quite strange. It looks like this is only misaligning the stars. This happens elsewhere too. Please drop this and similar parts of this commit (where blanks misalign the comments/code).